backend/apps/files/models.py

```diff
@@ -17,3 +20,27 @@ class File(models.Model):
 
     def __str__(self):
         return self.file_name
+
```

Comment:
redo all these models in the other related apps. consider these points below when doing 

1. PK is not needed since it's automatically handled by django 
2. All models should related to File model via the related relationship not as just a field 
3. check all the fields again